frontend/src/store.js

```diff
@@ -82,6 +86,7 @@ const store = createStore({
         }
       }
       res = await res.json();
+      commit('setConferenceRoomType', res.room.conferenceType)
```

Comment:
whats the reason to set this here? Just beacuse I join an SFU room should not change the setting in the Preferences.

Reply:
This is just to return the conferenceType so It can be displayed correctly to those who join the call. It doesn't change the preferences just displays the current sfu preference for the call.

Reply:
The store should match what the preferences are, at least that's how it is working currently.
If you want to display the type in the Room.vue then just take it from the room.